server/api/controllers/config/environments/environmentsConfigController.js

```diff
@@ -6,12 +6,13 @@ const KEY_NAME = 'EnvironmentName';
 
 let _ = require('lodash');
 let co = require('co');
+let Promise = require('bluebird');
```

Comment:
This is because the module makes use of the `Promise.map` feature of bluebird that is not in the ES6 standard.